llama-index-core/pyproject.toml

```diff
@@ -78,7 +78,7 @@ codespell = {extras = ["toml"], version = ">=v2.2.6"}
 docker = "^7.0.0"
 ipython = "8.10.0"
 jupyter = "^1.0.0"
-llama-cloud = ">=0.0.6"
+llama-cloud = ">=0.0.9"
```

Comment:
@logan-markewich does this need to be listed in the `llama-index-core/pyproject.toml` at all? seems like we need to update `llama-index-core` everytime we want to incorporate a new version of `llama-cloud` into `llama-index-indices-managed-llama-cloud`

Reply:
This shouldn't even need to be touched, since it's listed under dev dependencies (plus, its listed as >=, which should eliminate the issue no?)

Tbh I think somehow v0.10.54 of core isn't the correct version

Reply:
I can publish 0.10.54.post1 locally and see if that changes anything

Reply:
> This shouldn't even need to be touched, since it's listed under dev dependencies
yeah I would've thought that too, but then updating the llama-cloud version here fixed the error _(at least locally, which was the same error we're seeing in the CI)_

> (plus, its listed as >=, which should eliminate the issue no?)
maybe poetry tries to ensure that every version specified by the `>=` is compatible? im not sure if thats how it works